src/frontend/src/lib/components/dapps/DappTags.svelte

```diff
@@ -0,0 +1,18 @@
+<script lang="ts">
+	import { i18n } from '$lib/stores/i18n.store';
+	import { replacePlaceholders } from '$lib/utils/i18n.utils';
+
+	export let tags: string[] = [];
```

Comment:
Are you going to reuse this component elsewhere? If not, should we pass the dapp description and destructure the tags and dappme within the component to simplify its interface?

Reply:
I'm going to reuse it

Reply:
Oki doki, let's see then. Thanks for the feedback!